src/VisualStudio/Core/Def/Implementation/MoveStaticMembers/StaticMemberSelectionViewModel.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.LanguageServices.Implementation.Utilities;
+using Microsoft.VisualStudio.Utilities;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.MoveStaticMembers
+{
+    internal class StaticMemberSelectionViewModel : AbstractNotifyPropertyChanged
+    {
+        private readonly IUIThreadOperationExecutor _uiThreadOperationExecutor;
+        private readonly ImmutableDictionary<ISymbol, Task<ImmutableArray<ISymbol>>> _symbolToDependentsMap;
+        private readonly ImmutableDictionary<ISymbol, SymbolViewModel<ISymbol>> _symbolToMemberViewMap;
+
+        public StaticMemberSelectionViewModel(
```

Comment:
Not necessarily needed in this PR. But please make sure after all the UI details completes, add some ViewModel test. 
exmaple: https://sourceroslyn.io/#Microsoft.VisualStudio.LanguageServices.UnitTests/PullMemberUp/PullMemberUpViewModelTest.vb,a6d70fd2561a2e76